stardist/bioimageio_utils.py

```diff
@@ -343,6 +345,7 @@ def export_bioimageio(
     mode="tensorflow_saved_model_bundle",
     min_percentile=1.0,
     max_percentile=99.8,
+    generate_default_deps=False,
     overwrite_spec_kwargs=None,
```

Comment:
```suggestion
    overwrite_spec_kwargs=None,
    generate_default_deps=False,
```

Reply:
Oops, should have kept the order, thanks